docs/apache-airflow/core-concepts/auth-manager/simple.rst

```diff
@@ -31,23 +31,30 @@ the logic and implementation of the simple auth manager is **simple**.
 Manage users
 ------------
 
-Users are managed through the `webserver config file <https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#config-file>`__.
-In this file, the list of users are defined in the constant ``SIMPLE_AUTH_MANAGER_USERS``. Example:
+Users are managed through the Airflow configuration. Example:
 
-.. code-block:: python
+.. code-block:: ini
 
-  SIMPLE_AUTH_MANAGER_USERS = [
-      {
-          "username": "admin",
-          "role": "admin",
-      }
-  ]
+  [core]
+  simple_auth_manager_users = "bob,admin peter,viewer"
 
+or
+
+.. code-block:: bash
+
+  export AIRFLOW__CORE__SIMPLE_AUTH_MANAGER_USERS='bob,admin peter,viewer'
```

Comment:
nit: not sure we need to double up on the example. We don't generally duplicate the examples to show both options.